Makefile

```diff
@@ -53,15 +53,15 @@ all: dev server benchkv
 parser:
 	@echo "remove this command later, when our CI script doesn't call it"
 
-dev: checklist test check
+dev: checklist check test 
```

Comment:
I think `check` is usually take less time in the test, and usually easy to solve.
We should run `check` at first.

Reply:
But if we change this order,
because https://github.com/pingcap/tidb/issues/8947, we may hard to use `make dev` .( The most important part cannot be tested by `make dev`)